e2e-tests/recreate/compare/oldrecr-pood-descr

```diff
@@ -0,0 +1,163 @@
+Priority:     0
+Node:         gke-max-23890-default-pool-61907bb2-8lmh/10.156.15.212
+Start Time:   Mon, 05 Jul 2021 20:48:07 +0300
```

Comment:
So much dynamic data. Don't thing it's needed here

Reply:
https://github.com/percona/percona-postgresql-operator/pull/71/commits/6af9b2389e856fb97600ea59b222f3ae1b4d9a38